provisioning/miniprov/test/test_cdf.py

```diff
@@ -255,19 +345,26 @@ def ret_values(value: str) -> Any:
                 'server_node>MACH_ID>hostname':
                 'myhost',
                 'server_node>MACH_ID>name': 'mynodename',
-                'server_node>MACH_ID>storage>cvg[0]>data_devices': ['/dev/sdb'],
+                'server_node>MACH_ID>storage>cvg[0]>data_devices':
+                ['/dev/sdb'],
                 'server_node>MACH_ID>network>data>private_interfaces':
                 ['eth1', 'eno2'],
                 'server_node>MACH_ID>s3_instances':
                 1,
                 'server_node>MACH_ID>network>data>interface_type':
                 'o2ib',
-                'cluster>92f444df-87cc-4137-b680-aab3b35d1695>site>storage_set_count': 1,
-                'cluster>92f444df-87cc-4137-b680-aab3b35d1695>storage_set>server_node_count': 1,
-                'cluster>92f444df-87cc-4137-b680-aab3b35d1695>storage_set[0]>name': 'StorageSet-1',
-                'cluster>92f444df-87cc-4137-b680-aab3b35d1695>storage_set[0]>server_nodes': ['srvnode_1'],
-                'cluster>92f444df-87cc-4137-b680-aab3b35d1695>storage_set[0]>durability>data': 1,
-                'cluster>92f444df-87cc-4137-b680-aab3b35d1695>storage_set[0]>durability>parity': 0
+                'cluster>92f444df-87cc-4137-b680-aab3b35d1695>site>storage_set_count':
+                1,
+                'cluster>92f444df-87cc-4137-b680-aab3b35d1695>storage_set>server_node_count':
+                1,
+                'cluster>92f444df-87cc-4137-b680-aab3b35d1695>storage_set[0]>name':
+                'StorageSet-1',
+                'cluster>92f444df-87cc-4137-b680-aab3b35d1695>storage_set[0]>server_nodes':
+                ['srvnode_1'],
+                'cluster>92f444df-87cc-4137-b680-aab3b35d1695>storage_set[0]>durability>data':
```

Comment:
Should this path contain pooltype for e.g sns?

Reply:
Addressed: unused keys were removed from tests.